front/app/containers/Admin/reporting/components/ReportBuilder/Templates/ProjectTemplate/index.tsx

```diff
@@ -186,7 +186,7 @@ const ProjectTemplate = ({ reportId, projectId }: Props) => {
   if (enabled) {
     return <ProjectTemplateContent reportId={reportId} projectId={projectId} />;
   } else {
-    return <Element id="phase-report-template" is={Box} canvas />;
```

Comment:
Apparently this was breaking the idea template test. No idea this `id` mattered

Reply:
But the id was the same in this commit, where the e2e tests succeeded https://github.com/CitizenLabDotCo/citizenlab/pull/7427/commits/c71fde1b29344ffaa639d402ec6a422ce15a09de

Reply:
Or is the test flaky because of it?

Reply:
I think in the test previously, this code path was never reached, because `enabled` was determined by the `useEditor` call instead of the new `TemplateContext` (and we were still in the editor)